neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -279,6 +280,7 @@ def create_experiment(self,
                 currentBranch=git_info.active_branch,
                 repositoryDirty=git_info.repository_dirty
             )
+        entrypoint, source_target_pairs = get_source_code_to_upload(upload_source_files=upload_source_files)
```

Comment:
Why uploading source code was moved to create_experiment in backend? In most cases we had a 1-1 relation _one backend method - one api endpoint_. What was wrong with it?

Reply:
* First I wasn't sure how the solution will look like, so I've decided to make it similar to what we have in alpha client - so that's why I've created `get_source_code_to_upload`, which is "equivalent" of alpha `upload_source_code`.
* In the end of the day uploading files (`UploadFileSet`) as one of arguments passed to `executeOperations` when creating the experiment sounds reasonable to me.